### PR TITLE
Clarify opening terminal to repo directory

### PR DESCRIPTION
**Pull request by anthonyj33**
Added clarification to the steps for running the code locally. Explains that the terminal must be open to the same directory as the repository.

### CODE REVIEW

**Comment by Awesome-E** [comment]
"and use the built-in terminal" sounds like you're referring to the system's terminal. I wouldn't remove it, but maybe reword the whole sentence?

**Comment by anthonyj33** [comment]
Good point. I reworded it slightly for clarity. Let me know how it sounds.

**Comment by Awesome-E** [comment]
That line is quite long but I think it's fine

**Review by Awesome-E — APPROVED** [review]
